packages/bpk-component-price-range/src/BpkPriceRange-test.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render } from '@testing-library/react';
+
+import BpkPriceRange from './BpkPriceRange';
+
+describe('BpkPriceRange', () => {
+  const segments = {
+    low: {
+      price: '£100',
+      percentage: 20,
+    },
+    high: {
+      price: '£200',
+      percentage: 80,
+    },
+  };
+
+  it('should render low price indicator correctly', () => {
+    const { asFragment } = render(
+      <BpkPriceRange
+        marker={{ price: '£50', percentage: 10 }}
+        segments={segments}
+      />,
+    );
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
Snapshots can be unreliable and we ideally move away from these.

Would you be able to modify these tests to check if the elements are in the document instead of snapshots?

Reply:
Great, I have add some assertions